### ISSUE TITLE
Niet meer op google play / geen apk download

### ISSUE THREAD

**Issue by Vosjedev** [Argo-Client/App#17]
Ik kan de versie op google play niet meer vinden, en de links op de site en de readme sturen naar een niet bestaande app.

Is het mischien handig om van de releases een apk te attachen? Waarom is de app van google play verdwenen?

**Comment by Netfloex** [comment]
Voor google play moesten we geloof ik wat moeilijke dingen doen (SDK updaten ofzo) en werkt daarom niet meer.
De website zou het wel moeten doen...
Ik heb even de workflow opnieuw gerunned, dus hopelijk komt er binnenkort een vers apktje uit gerold.

**Comment by Vosjedev** [comment]
Ah, ik heb inderdaad een artifact (met apks) gevonden in [de nieuwse action](https://github.com/Argo-Client/App/actions/runs/9094906753), dus ik kan in ieder geval de app installeren. Maar als de downloads op de website onderaan moeten zitten werken die op mijn browser (nieuwse firefox) helaas niet...
![image](https://github.com/Argo-Client/App/assets/100359465/ec0a191b-a280-4e54-a73a-341b04b34c48)
Ik zie hier een `Downloads` textje met een logo erboven, doen allebij niks. De `Previous` en `Next` buttons doen ook niks...

**Comment by Netfloex** [comment]
Klopt, er is weer eens iets mis met de site, word hopelijk binnenkort gefixed

**Comment by Vosjedev** [comment]
Ah, kan gebeuren.

**Comment by Mrmda** [comment]
Is dit al gelukt?
